submissions/AndriiZahumnyi/js-dom/script/index.js

```diff
@@ -0,0 +1,39 @@
+
+const hamburgerToggle = document.querySelector('#menu-toggle');
+
+const flexMenu = document.querySelector('.flex-menu');
+
+hamburgerToggle.addEventListener('click', function (e) {
```

Comment:
You can simplify this handler using arrow functions:
```suggestion
hamburgerToggle.addEventListener('click', () => flexMenu.classList.toggle('active'));
```

Reply:
Also, you can move adding this event listener to the bottom of your script and keep your event listeners in one place. It can help other developers and you to navigate your code easier.